.github/workflows/package.yml

```diff
@@ -0,0 +1,36 @@
+name: Package
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  Package:
+    name: Package and Publish Docker Image
+    if: github.event.base_ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Get Version
+      id: get_version
+      run: |
+        echo ::set-output name=version::${GITHUB_REF/refs\/tags\/}
+  
+    - name: Checkout Branch
+      uses: actions/checkout@v2
+
+    - name: Build
+      run: |
+        docker build --build-arg DEPLOY_VERSION=${{ steps.get_version.outputs.version }} -t docker.io/prebid/prebid-cache:${{ steps.get_version.outputs.version }} .
+
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_HUB_USERNAME }}
```

Comment:
This will require us to create repo level secrets in Github with the Prebid account username and access token